Required to build on Cygwin due to a bug (?) in Cygwin (specifically newlib)
whereas the removed lines themselves were added apparently to fix building on
BSDs:

https://github.com/dlitz/pycrypto/commit/574ffac9f5d290933e8c0678c430fa8eabc962d0
diff -u a/setup.py b/setup.py
--- a/setup.py	2013-10-14 23:38:10.000000000 +0200
+++ b/setup.py	2016-07-06 15:10:54.271597100 +0200
@@ -114,13 +114,6 @@
 
         # Tweak compiler options
         if self.compiler.compiler_type in ('unix', 'cygwin', 'mingw32'):
-            # Tell GCC to compile using the C99 standard.
-            self.__add_compiler_option("-std=c99")
-
-            # ... but don't tell that to the aCC compiler on HP-UX
-            if self.compiler.compiler_so[0] == 'cc' and sys.platform.startswith('hp-ux'):
-                self.__remove_compiler_option("-std=c99")
-
             # Make assert() statements always work
             self.__remove_compiler_option("-DNDEBUG")
 
Common subdirectories: a/src and b/src
